rust_src/src/casetab.rs

```diff
@@ -65,18 +231,81 @@ pub fn standard_case_table() -> LispObject {
 ///  in which case it is deduced from CANONICALIZE.
 #[lisp_fn(name = "set-case-table", c_name = "set_case_table")]
 pub fn set_case_table_lisp(table: LispObject) -> LispObject {
-    unsafe { set_case_table(table, false) }
+    set_case_table(table, false)
 }
 
 /// Select a new standard case table for new buffers.
 /// See `set-case-table' for more info on case tables.
 #[lisp_fn]
 pub fn set_standard_case_table(table: LispObject) -> LispObject {
-    unsafe { set_case_table(table, true) }
+    set_case_table(table, true)
+}
+
+#[no_mangle]
+pub unsafe extern "C" fn init_casetab_once() {
+    def_lisp_sym!(Qcase_table, "case-table");
+    Fput(Qcase_table, Qchar_table_extra_slots, 3.into());
```

Comment:
`put` has been ported (although it would require casting the symbols to `LispSymbolRef`).

Reply:
Exactly. I did not see the benefit of the Symbol transform here. :shrug: